sierra/src/main/java/org/httprpc/sierra/DatePicker.java

```diff
@@ -175,8 +175,6 @@ void updateMonth() {
                     date = date.plus(1, ChronoUnit.DAYS);
                 }
             }
-
```

Comment:
I don't remember why I added this, but the code appears to work fine without it.